jcl/src/openj9.jvm/share/classes/com/ibm/jvm/CRIUSupport.java

```diff
@@ -42,5 +112,34 @@ private CRIUSupport() {}
 	public static boolean isCRIUSupportEnabled() {
 		return criuSupportEnabled;
 	}
-	
+
+	/**
+	 * Checkpoint the JVM
+	 * 
+	 * @param checkPointDir the directory in which the checkpoint data will be stored
+	 * 
+	 * @return return CRIUResult
```

Comment:
Pls document the SecurityManager requirements.

Reply:
Typically there is at least a `@throws SecurityException` with details.

Reply:
Since we are catching all other exceptions and storing it in CRIUResult, for consistency should we do that here as well?